diff --git a/dist/src/index.js b/dist/src/index.js
index 53d21bd957dd4d3cb6ccd6c67a51d9dab137bd4c..de82ef78851949f8e8aabad9c0133e04f35a892f 100644
--- a/dist/src/index.js
+++ b/dist/src/index.js
@@ -131633,10 +131633,7 @@ class HumanFriendlyWriter {
             process.stdout.clearScreenDown();
         }
         if (this.isVerbose) {
-            this.stream.write('\n');
-            this.stream.write('====================================\n');
-            this.stream.write(`Loading svelte-check in workspace: ${workspaceDir}`);
-            this.stream.write('\n');
+            this.stream.write(`Loading svelte-check in workspace: ${workspaceDir}\n`);
             this.stream.write('Getting Svelte diagnostics...\n');
             this.stream.write('\n');
         }
@@ -131650,7 +131647,7 @@ class HumanFriendlyWriter {
             let msg;
             if (this.isVerbose) {
                 const code = this.formatRelatedCode(diagnostic, text);
-                msg = `${diagnostic.message} ${source}\n${pc.cyan(code)}`;
+                msg = `${diagnostic.message} ${source}\n${pc.cyan(code.trimEnd())}`;
             }
             else {
                 msg = `${diagnostic.message} ${source}`;
@@ -131687,7 +131684,6 @@ class HumanFriendlyWriter {
         return text.substring(src.offsetAt({ line, character: 0 }, text), src.offsetAt({ line, character: Number.MAX_SAFE_INTEGER }, text));
     }
     completion(_f, errorCount, warningCount, fileCountWithProblems) {
-        this.stream.write('====================================\n');
         const message = [
             'svelte-check found ',
             `${errorCount} ${errorCount === 1 ? 'error' : 'errors'} and `,
